gradle-plugin/src/main/kotlin/dev/shreyaspatil/composeCompilerMetricsGenerator/plugin/task/ComposeCompilerReportGenerateTask.kt

```diff
@@ -45,17 +49,23 @@ abstract class ComposeCompilerReportGenerateTask : DefaultTask() {
     @get:Input
     abstract val compileKotlinTasks: Property<String>
 
-    private val reportExtension: ComposeCompilerReportExtension
-        get() = ComposeCompilerReportExtension.get(project)
```

Comment:
I have a question here: Why we removed this? Can't we just get these params from this extension?

Reply:
Oh, I get it. We created individual input and output fields.